pywbemtools/pywbemcli/_cmd_class.py

```diff
@@ -1076,21 +1080,28 @@ def get_default_ns(default_ns):
         return default_ns if not isinstance(namespaces, tuple) else [default_ns]
 
     # Return the provided namespace(s) by expanding each entry that has
-    # comma-separated values
+    # comma-separated values add adding those without comma
+
     ns_names = []
     if namespaces:
-        for ns in namespaces:
-            ns_names.extend(ns.split(','))
-        return ns_names
+        if isinstance(namespaces, (tuple)):
```

Comment:
I suggest removing the parenthesis around `tuple` since they don't do anything.

Reply:
DONE